waterdata/templates/base.html

```diff
@@ -8,6 +8,7 @@
         <link rel="stylesheet" href="{{ config.STATIC_ROOT }}/main.css">
         <script type="text/javascript">var SERVICE_ROOT = '{{ config.SERVICE_ROOT }}/nwis';</script>
         <script async src="{{ config.STATIC_ROOT }}/bundle.js"></script>
+        {% block linked_data %}{% endblock %}
```

Comment:
Consider making this a more general tag ... maybe something like {% block page_script %}. Then its clear it could be used anything really... page specific javasscript, json ld, etc.